tests/phpunit/tests/cron.php

```diff
@@ -1112,4 +1112,58 @@ function() {
 		$this->assertSame( 'could_not_set', $unscheduled->get_error_code() );
 	}
 
+	/**
+	 * If the option "cron" is empty _get_cron_array should return false!!
+	 *
+	 * @ticket 53651
+	 * @covers ::_get_cron_array
+	 */
+	public function test__get_cron_array(){
+		$cron = get_option( 'cron' );
+		unset( $cron['version'] );
+		$this->assertEquals( _get_cron_array(), $cron );
```

Comment:
```suggestion
		$this->assertSame( _get_cron_array(), $cron );
```

`assertEquals()` is a loose type comparison, while `assertSame()` is a type safe comparison.